### PR TITLE
Update ports doc.

### PR DESCRIPTION
**Pull request by erictune**

### CODE REVIEW

**Comment by jlowdermilk** [comment]
LGTM, but needs rebase, `make clean` and `hack/run-gendocs.sh > docs/kubectl.md` to pick up changes from #4355.

**Comment by erictune** [comment]
rebased

File: docs/accessing_the_api.md [hunk 1]
```diff
@@ -48,16 +48,18 @@ variety of uses cases:
    3. Scheduler and Controller-manager processes, which need to do read-write
       API operations.  Currently, these have to run on the 
       operations on the apiserver.  Currently, these have to run on the same
-      host as the apiserver and use the Localhost Port.
+      host as the apiserver and use the Localhost Port.  The
```
Comment (thockin) [inline docs/accessing_the_api.md]: Weird trailing "the"

**Comment by erictune** [comment]
ready

**Comment by jlowdermilk** [comment]
Note, shippable failure looks like flake. Will merge on green travis.

**Pull request merged**